content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -91,73 +103,89 @@ For example:
 
-```console
-GET /api/v1/namespaces/test/pods?watch=1&resourceVersion=10245&allowWatchBookmarks=true
----
-200 OK
+For example:
+
+```
+GET /api/v1/namespaces/test/pods?watch=1&resourceVersion=10245&allowWatchBookmarks=true HTTP/1.1
+…more headers…
+```
+```
+HTTP/1.1 200 OK
 Transfer-Encoding: chunked
 Content-Type: application/json
 
 {
   "type": "ADDED",
   "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "10596", ...}, ...}
 }
-...
+…
 {
   "type": "BOOKMARK",
   "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "12746"} }
 }
 ```
 
-`Bookmark` events can be requested by `allowWatchBookmarks=true` option in watch requests, but clients shouldn't assume bookmarks are returned at any specific interval, nor may they assume the server will send any `bookmark` event.
+You can request `bookmark` events by setting `allowWatchBookmarks=true` when making a Watch request. Clients shouldn't assume bookmarks are returned at any specific interval, nor may they assume the server will send any `bookmark` event.
 
 ## Retrieving large results sets in chunks
 
 {{< feature-state for_k8s_version="v1.9" state="beta" >}}
 
-On large clusters, retrieving the collection of some resource types may result in very large responses that can impact the server and client. For instance, a cluster may have tens of thousands of pods, each of which is 1-2kb of encoded JSON. Retrieving all pods across all namespaces may result in a very large response (10-20MB) and consume a large amount of server resources. Starting in Kubernetes 1.9 the server supports the ability to break a single large collection request into many smaller chunks while preserving the consistency of the total request. Each chunk can be returned sequentially which reduces both the total size of the request and allows user-oriented clients to display results incrementally to improve responsiveness.
+On large clusters, retrieving the collection of some resource types may result in very large responses that can impact the server and client. For instance, a cluster may have tens of thousands of pods, each of which is 1-2KiB of encoded JSON. Retrieving all pods across all namespaces may result in a very large response (10-20MB) and consume a large amount of server resources. The kube-apiserver supports the ability to break a single large collection request into many smaller chunks while preserving the consistency of the total request. Each chunk can be returned sequentially; this chunking reduces the size of each HTTP response message and also allows user-oriented clients to display results incrementally to improve responsiveness. Splitting retrieval over multiple HTTP responses means that if a client needs to retry a fetch, only the missing portion needs to be retransmitted.
 
```

Comment:
What is a "user-oriented" client -- a web browser?
Possible edit?
Each chunk is returned sequentially. Chunking reduces the size of an HTTP response which improves responsiveness in user-oriented clients. When a message is split over multiple HTTP responses and the client retries a fetch, only the missing portion of the message needs to be retransmitted by the API server.

Reply:
> What is a "user-oriented" client -- a web browser?

Let's save dealing with that to another PR? This one is already quite involved. I'd rather make _some_ improvements now than hold off making any until I fully understand the Kubernetes API.